app/styles/_alt-channel.scss

```diff
@@ -18,6 +18,19 @@
     letter-spacing: 2px;
     text-transform: uppercase;
   }
+
+  .list {
+    padding: 30px 0;
+
+    &:after {
+      content: " ";
+      width: 25px;
+      height: 40px;
```

Comment:
I see you changed it, but specifically shouldn't this be 100% so that if the height of the list changes (maybe we use a bigger font size), this thing will adjust accordingly?